### ISSUE TITLE
<Accordion/> - 'open' property of AccordionItem isn't dynamically updated

### ISSUE THREAD

**Issue by Hadar-Grossman** [wix/wix-style-react#4967]
# 🐛 Bug Report

### 🏗 Relevant Components

Accordion
AccordionItem

### 😯 Current Behavior

```jsx
<Accordion
      multiple
       items={[
                { title: 'first', icon: <ChevronRight />, children: (<div>First!!!</div>), open: isOpen },
               { title: 'second', icon: <ChevronRight />, children: (<div>Second!!!</div>), open: isOpen }
       ]}
 />
```

```javascript
isOpen = false; // Inside the component's constructor

isOpen = true// After a certain event was triggered
```
The items of the Accordion remain closed.

### 🤔 Expected Behavior

Same scenario but the Accordion items are open after changing 'isOpen' value to 'true'.

### 👣 Steps to Reproduce

Create an Accordion as described above.

  - **WSR version**: `^7.0.1`
  - **Yoshi version**: `^4.11.2`

### 👀 Severity

Major

**Comment by bamai** [comment]
Reproduce the bug:
```
class MyComponent extends React.Component {
  
  state={isOpen: false} 
  
  render() {
  const { isOpen } = this.state;
    return (
    <>
      <Accordion
        multiple
        items={[
        {
          title: 'First Row With Button',
          open: isOpen,
          children: (
          <Text>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
            eiusmod tempor incididunt ut labore et dolore magna aliqua
          </Text>
          ),
          buttonType: 'button',
          expandLabel: 'Show More',
          collapseLabel: 'Less',
        },
        {
          title: 'Second Row With Icon',
          open: isOpen,
          children: (
          <Text>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
            eiusmod tempor incididunt ut labore et dolore magna aliqua
          </Text>
          ),
          icon: <Icons.InfoCircle />,
          expandLabel: 'Show More',
          collapseLabel: 'Less',
        },
      ]}/>
      <Button onClick={()=> {this.setState({isOpen: !isOpen})}}>toggle</Button>
    </>);
  }
}
```

**Comment by bamai** [comment]
Hi @Hadar-Grossman,
I discussed with our UX and they wanted to know what do you need this functionality for. Do you have a design flow or Zeplin link to share?
Thanks

**Comment by Hadar-Grossman** [comment]
Hi @bamai, this is the design flow that requires this kind of behaviour - 
https://app.zeplin.io/project/5e1b105927b9f9abc44b392a/screen/5e1b13a036c30c7dfbb70db0

**Comment by bamai** [comment]
Hi @Hadar-Grossman,
`items` is a prop which receives an array of objects. When changing a prop of an object, the **pointer** to the object (and array) doesn't change. Therefore, the `Accordion` doesn't re-render. This is React behaviour.

**Comment by Hadar-Grossman** [comment]
Any progress regarding this bug ? @EgoziE @bamai

**Comment by shlomitc** [comment]
We'll look into it next week
